docs/rules/padded-blocks.md

```diff
@@ -18,8 +18,12 @@ padded blocks or never do it.
 
 This rule enforces consistent padding within blocks.
 
-This rule takes one argument. If it is `"always"` then blocks must start **and** end with a blank line. If `"never"`
-then all blocks should never start **or** end with a blank line. The default is `"always"`.
+This rule takes one argument. If it is `"always"` (the default) then block statements must start **and** end with a blank line. If `"never"`
+then all block statements should never start **or** end with a blank line. By default, this rule ignores padding in switch statements.
```

Comment:
While we're here, the wording "If `"never"` then all block statements should start **or** end with a blank line." seems awkward to me. Would you mind changing it to "If `"never"`, then block statements should neither start nor end with a blank line."? That seems cleaner to me.